plugins/become/machinectl.py

```diff
@@ -66,15 +66,37 @@
             ini:
               - section: machinectl_become_plugin
                 key: password
+    notes:
+      - This plugin requires a polkit rule which will alter the behaviour of machinectl to ask directly for the user
```

Comment:
Does this mean that this plugin will stop working for users which don't have that polkit rule and for whom this worked before? If yes, this is a breaking change and cannot be merged.

Reply:
No it shouln't stop working because, the plugin only really worked when you are root. If you used the plugin without root it always failed because the prompt was not handled by ansible. With this the prompt will work when the user is directly asked for a password. But the normal prompt from polkit for machinectl would be to ask first for a member of for which the admin rule matched, and the for the password. But since I don't want to add new options to allow the user to be selected by ansible, the polkit rule is needed.